### PR TITLE
[enterprise-4.11] RN entry for BZ2102230

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.11

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2102230

Link to docs preview: http://file.rdu.redhat.com/jdohmann/BZ2102230-rn/release_notes/ocp-4-11-release-notes.html#ocp-4-11-rhcos-redirector-url

Additional information: RN entry for #48037

### CODE REVIEW

**Comment by mike-nguyen** [comment]
I don't think customers can browser `rhcos.mirror.openshift.com`. It is a redirector to the S3 backend so customers will need to know the exact URL to the image they are looking for.

**Comment by jldohmann** [comment]
@mike-nguyen good point 😅 I've updated the text and included a link to the relevant doc, could you PTAL again?

273ed5c..19ce228
19ce228..3e22fc7

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -65,6 +65,11 @@ This release adds improvements related to the following components and concepts.
 
 {op-system} now uses {op-system-base-full} 8.6 packages in {product-title} 4.11 and above. This enables you to have the latest fixes, features, and enhancements, as well as the latest hardware support and driver updates.
 
+[id="ocp-4-11-rhcos-redirector-url"]
+==== Updated {op-system} registry URL
+
+The redirector host name for downloading {op-system} boot images is now `rhcos.mirror.openshift.com`. You must configure your firewall to grant access to this registry. See xref:../installing/install_config/configuring-firewall.html#configuring-firewall_configuring-firewall[Configuring your firewall for {product-title}] for more information.
```
Comment (mike-nguyen) [inline release_notes/ocp-4-11-release-notes.adoc:71]:
```suggestion
The redirector host name for downloading {op-system} boot images is now `rhcos.mirror.openshift.com`. You must configure your firewall to grant access to the boot images. See xref:../installing/install_config/configuring-firewall.html#configuring-firewall_configuring-firewall[Configuring your firewall for {product-title}] for more information.
```

Comment (mike-nguyen) [reply release_notes/ocp-4-11-release-notes.adoc:71]: Its more like a file server than a registry.

**Review by mike-nguyen — CHANGES_REQUESTED** [review]

**Review by travier — COMMENTED** [review]
Same comment as Mike but otherwise LGTM

File: release_notes/ocp-4-11-release-notes.adoc (hunk 1 above)
Comment (jldohmann) [reply release_notes/ocp-4-11-release-notes.adoc]: thanks, Mike. Fixed 👍

**Comment by jldohmann** [comment]
3e22fc7..db38682 implement feedback

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -65,6 +65,11 @@ This release adds improvements related to the following components and concepts.
 
 {op-system} now uses {op-system-base-full} 8.6 packages in {product-title} 4.11 and above. This enables you to have the latest fixes, features, and enhancements, as well as the latest hardware support and driver updates.
 
+[id="ocp-4-11-rhcos-redirector-url"]
+==== Updated {op-system} registry URL
+
+The redirector host name for downloading {op-system} boot images is now `rhcos.mirror.openshift.com`. You must configure your firewall to grant access to the boot images. See xref:../installing/install_config/configuring-firewall.html#configuring-firewall_configuring-firewall[Configuring your firewall for {product-title}] for more information.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-11-release-notes.adoc:71]: Recommend spelling out RHCOS in the first sentence
hostname one word per ISG
Consider putting "For more information," first, as that's preferred by the ISG

```suggestion
The redirector hostname for downloading {op-system-first} boot images is now `rhcos.mirror.openshift.com`. You must configure your firewall to grant access to the boot images. For more information, see xref:../installing/install_config/configuring-firewall.adoc#configuring-firewall_configuring-firewall[Configuring your firewall for {product-title}].
```

**Comment by jldohmann** [comment]
db38682..1d9ee4d implement peer review feedback

**Comment by opayne1** [comment]
lgtm!

**Pull request merged**